tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -549,6 +1078,60 @@ static void test_change_microphone_gain(void)
 	printk("Microphone gain changed\n");
 }
 
+static void test_broadcast_reception_start(size_t accepter_count)
+{
+	struct bt_cap_commander_broadcast_reception_start_param reception_start_param;
+	struct bt_cap_commander_broadcast_reception_start_member_param param[CONFIG_BT_MAX_CONN];
+	int err;
+
+	reception_start_param.type = BT_CAP_SET_TYPE_AD_HOC;
+	reception_start_param.count = accepter_count;
+	reception_start_param.param = param;
+
+	for (size_t i = 0; i < accepter_count; i++) {
+		reception_start_param.param[i].member.member = connected_conns[i];
+		bt_addr_le_copy(&reception_start_param.param[i].addr, &broadcaster_addr);
+		reception_start_param.param[i].adv_sid = broadcaster_info.sid;
+		reception_start_param.param[i].pa_interval = broadcaster_info.interval;
+		reception_start_param.param[i].broadcast_id = broadcaster_broadcast_id;
+		reception_start_param.param[i].num_subgroups = 0;
+		err = bt_bap_base_foreach_subgroup((const struct bt_bap_base *)received_base,
+						   add_pa_sync_base_subgroup_cb, &param[i]);
+		if (err != 0) {
+			FAIL("Could not populate subgroup information: %d\n", err);
+		}
+	}
+
+	err = bt_cap_commander_broadcast_reception_start(&reception_start_param);
+	if (err != 0) {
+		FAIL("Could not initiate broadcast reception start: %d\n", err);
+		return;
+	}
+
+	WAIT_FOR_FLAG(flag_broadcast_reception_start);
+
+	backchannel_sync_wait(2); /* wait for the acceptor to receive data */
+
+	backchannel_sync_wait(2); /* wait for the acceptor to receive a metadata update */
+}
+
+static void test_broadcast_reception_stop(size_t accepter_count)
+{
+	struct bt_cap_commander_broadcast_reception_stop_param reception_stop_param;
+	int err;
+
+	/* reception stop is not implemented yet, for now the following command will fail*/
+	reception_stop_param.type = BT_CAP_SET_TYPE_AD_HOC;
+	reception_stop_param.members = NULL;
+	reception_stop_param.count = 0U;
+	err = bt_cap_commander_broadcast_reception_stop(&reception_stop_param);
+	if (err != 0) {
+		printk("Command not implemented yet, could not stop broadcast reception %d\n", err);
+	}
+
+	backchannel_sync_wait(2); /* wait for the acceptor to stop reception */
```

Comment:
This isn't related to the reception stop procedure. Suggest to move out of this function